app/models/spotlight/page.rb

```diff
@@ -7,7 +7,9 @@ class Page < ActiveRecord::Base
     MAX_PAGES = Spotlight::Engine.config.max_pages
 
     extend FriendlyId
-    friendly_id :title, use: %i[slugged scoped finders history], scope: %i[exhibit locale]
+    friendly_id :title, use: %i[slugged scoped finders history], scope: %i[exhibit locale] do |config|
```

Comment:
We defined friendly_id on the `AboutPage` and `FeautrePage` classes as well so we would need to change it there too.   That means that we may only need to reserve `contacts` for AboutPages (🤷‍♂️).

We called this out in https://github.com/projectblacklight/spotlight/pull/2080#discussion_r199303425 but perhaps now would be a good time to validate that is still necessary.

Reply:
Nope, probably still true. I'll leave a code comment  here as a reminder too.